### DISCUSSION TITLE
Multiple picker field uranium ui problem color doesn't change and the background color of the options

### DISCUSSION THREAD

**Discussion by rensu31** [enisn/UraniumUI#675] [General]
Hi my problem is that multiple picker field is that when i click for the options it only shows like this 

![Prob](https://github.com/enisn/UraniumUI/assets/143232714/c7d1275b-e644-463b-a2c7-942ecda9bfb3)

i cant see the options and i have tried everything like background color textcolor it still doesnt work pls help me

 `
                    <ur:MultiplePickerField
                        x:Name="Addon"
                        Title="Pick some options"
                        AccentColor="White"
                        BackgroundColor="White"
                        TitleColor="Black">
                        <material:MultiplePickerField.ItemsSource>
                            <x:Array Type="{x:Type x:String}">
                                <x:String>Pearls</x:String>
                                <x:String>Whipped Cream</x:String>
                                <x:String>Choco Syrup</x:String>
                                <x:String>Crushed Oreos</x:String>
                            </x:Array>
                        </material:MultiplePickerField.ItemsSource>
`

**Comment by rensu31** [comment]
@enisn

**Reply by enisn** [reply]
Have you imported Styles from Material?

Make sure you followed Getting Started documentation properly: ->https://enisn-projects.io/docs/en/uranium/latest/Getting-Started#existing-projects
![image](https://github.com/enisn/UraniumUI/assets/23705418/a9791474-a48a-4f1f-84a8-5aba71f457b4)

---

Same problem from here: https://x.com/EnisNecipoglu/status/1790799212508033426
